template-tags.php

```diff
@@ -417,19 +417,45 @@ function coauthors_ids( $between = null, $betweenLast = null, $before = null, $a
 	), null, $echo );
 }
 
-function get_the_coauthor_meta( $field ) {
+/**
+ * Outputs the co-authors Meta Data
+ *
+ * @param string $field Required The user field to retrieve.[login, email, nicename, display_name, url, type]
+ */
+function get_the_coauthor_meta( $field ) { 
 	global $wp_query, $post;
 
 	$coauthors = get_coauthors();
 	$meta = array();
-
+        
 	foreach ( $coauthors as $coauthor ) {
+                
 		$user_id = $coauthor->ID;
-		$meta[ $user_id ] = get_the_author_meta( $field, $user_id );
+                if ( isset($coauthor->data) ){
+                     $coauthor = $coauthor->data;
+                }
+                
+                if( $field == 'url' && isset ($coauthor->website) )
```

Comment:
>So, to kill two birds with one stone, we can do a check to see if the type is set and if it is, do another check to make sure it's the guest author type. Once all those checks pass, we can set the field to website from an input of the field user_url. 

I'm not sure I follow. Perhaps an example would help? (Also, it's possible this is clear if you're more familiar with the code).

One suggestion on the above is I'd create a new paragraph for the yoda and strict comparison to denote it's separate from the first item.

Reply:
Something like starting the check off with `if ( isset( $coauthor->type ) && 'user_url' === $field )`, then in that if statement for failsafe, throw in another `if ( 'guest-author' === $coauthor->type )` check.  Once all checks pass, we can assign `$field = 'website'` since that's the cap property equivalent of `user_url`.